apps/webapp/src/components/RootLayout/index.tsx

```
@@ -63,87 +63,71 @@
+        <nav
+          className={cn(
+            "relative hidden h-screen border-r lg:block w-72",
+            "bg-muted/50"
+          )}
+        >
+          <div className='flex items-center justify-center min-w-full'>
+            <div className="hidden lg:block pt-5">
               <Link href='/'>
-                {theme == "light" ? <img src="/logo-panora-black.png" className='w-14' /> : <img src="/logo-panora-white-hq.png" className='w-14' />}
+                {theme == "light" ? 
+                  <img src="/logo-panora-black.png" className='w-14' /> : 
```

Biome (1.9.4)

[error] Provide a text alternative through the alt, aria-label or aria-labelledby attribute

Meaningful alternative text on elements helps users relying on screen readers to understand content's purpose within a page.
If the content is decorative, redundant, or obscured, consider hiding it from assistive technologies with the aria-hidden attribute.

(lint/a11y/useAltText)